gum/guminterceptor.c

```diff
@@ -819,6 +822,15 @@ _gum_function_context_on_enter (FunctionContext * function_ctx,
       invocation_ctx->backend->data = &state;
 
 #ifdef HAVE_QNX
+    gpointer stack_address = &stack_address;
+    if (interceptor_ctx->thread_side_stack < stack_address &&
```

Comment:
Style-wise we tend to do it like this:

```
    if (stack_address >= interceptor_ctx->thread_side_stack &&
```

If “this thing we're looking at now” is greater or equal to "that thing that defines some boundary".